cmd/snaptel/flags.go

```diff
@@ -49,10 +49,11 @@ var (
 		Name:  "running",
 		Usage: "Shows running plugins",
 	}
-	flPassword = cli.BoolFlag{
```

Comment:
In my point of view, the issue is #1669 it's not valid - trying to pass a string when bool is expected should return an error. Please, notice how the usage is formed: "Require password for REST API authentication". Set this to true cause that user will be asked to provide the password - [here](https://github.com/intelsdi-x/snap/pull/1671/files#diff-437ff6f8535e7c3766454ed3a7568823L129). I do not think that this mechanism should be changed at the moment and in this way. @candysmurf, could we come back and trigger discussion about another idea(s) and bringing benefits? If you agree, feel free to open a new issue to make a space for such discussion (but as an RFC, not as identified bug) with providing use cases. Thank You